docs/software/programming.md

```diff
@@ -30,69 +32,71 @@ Python is a text based interpreted programming language with object oriented pro
 
 Source: By [www.python.org](http://www.python.org/community/logos/), [GPL](https://commons.wikimedia.org/w/index.php?curid=34991637)
 
-The install option explicitly only installs Python 3.  
-The `pip` resp. `pip3` Python package manager and development headers are included.
+The install option explicitly installs only **Python 3**.  
+The `pip` and `pip3` Python package manager and development headers are included.
 
 To use the `pip` package manager, a typical usage will be `pip3 install -U <module>`.
 
 ***
 
-Wikipedia: <https://wikipedia.org/wiki/Python_(programming_language)>
+- Python website: <https://www.python.org>
+- Python documentation, from beginner to advanced: <https://www.python.org/doc/>
 
 ## Docker
 
-Docker is used to build, ship, and run distributed applications.
-
-<!-- ![Docker logo](../assets/images/dietpi-software-programming-docker1.svg){: width="200px"} -->
+In 2013, Docker introduced containers. These are a standardized unit of software that allows developers to isolate their application from the environment. Docker is de facto standard to build and share containerized apps - from single bord computers (SBC), to desktop or cloud.
 
-![Docker functional block diagram](../assets/images/dietpi-software-programming-docker2.svg){: width="400px"}
+A Docker container image is a lightweight, standalone, executable package of software that includes everything needed to run an application: code, runtime, system tools, system libraries and settings.
 
-Source: [User:`Maklaan` - Based on a Docker blog post](https://commons.wikimedia.org/w/index.php?curid=37965701)
+=== "How to access the logs"
 
-=== "Access logs"
-
-    Access logs can be viewed with this command:
+    The Docker is available after the installation. Docker logs can be accessed using next command:
 
     ```sh
     journalctl -u docker -u containerd
     ```
 
-=== "Config files"
+=== "Configuration files"
 
-    The Docker configuration files are:
+    The location of the Docker configuration files:
 
     - Docker: `/etc/docker/daemon.json`
     - containerd: `/etc/containerd/config.toml`
 
 ***
 
-Website: <https://docs.docker.com/get-started/overview>  
-Configuration file: <https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file>  
-Logging: <https://docs.docker.com/config/containers/logging/configure>  
-Wikipedia: <https://wikipedia.org/wiki/Docker_(software)>
+- Docker website: <https://docs.docker.com/get-started/overview>  
+- Configuration file: <https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file>  
+- Logging: <https://docs.docker.com/config/containers/logging/configure>  
```

Comment:
On all other software pages we do not use lists. Do you want to change that intentionally?

Also was the Wikipedia linked here and the license below removed intentionally? I find both useful actually, just added links for the open-source licenses to the camera docs: https://github.com/MichaIng/DietPi-Docs/pull/380/files#diff-94991dd563cf0e88d0494bc5e7a7d08c6319079e44f40d5718c884b8d9302235R63

Reply:
Sure, I will re-enable them.